py-polars/tests/test_series.py

```diff
@@ -279,3 +279,21 @@ def test_iter():
     assert iter.__next__() == 2
     assert iter.__next__() == 3
     assert sum(s) == 6
+
+
+def test_describe():
+    num_s = pl.Series([1, 2, 3])
+    str_s = pl.Series(["a", "b", "c"])
+
+    assert num_s.describe() == {
+        "min": 1,
+        "max": 3,
+        "sum": 6,
+        "mean": 2.0,
+        "std": 0.816496580927726,
+        "count": 3,
+    }
+    try:
```

Comment:
Probably better to write
```py
with pytest.raises(TypeError) as exc_info:
    str_s.describe()
assert str(exc_info) == "This type is not supported"
```

Reply:
And you probably need to do the same with empty serie and `ValueError`

Reply:
Thanks for the review and the code snippet! I have added the additional tests